app/views/agent/confirmClientVrn.scala.html

```diff
@@ -27,11 +27,13 @@ <h1 class="heading-large">
         @messages("confirmClientVrn.heading")
     </h1>
 
-    <h2 class="bold-medium">@messages("confirmClientVrn.name")</h2>
-    <p>@customerDetails.clientName</p>
+    <dl>
+            <dt class="bold-medium bottom-padding">@messages("confirmClientVrn.name")</dt>
```

Comment:
Tag < dt > is aligned too far away from < dl >

Reply:
I have re-aligned as requested.